products/compute/api.yaml

```diff
@@ -15461,6 +15466,315 @@ objects:
+                            name: 'responseHeadersToAdd'
+                            description: |
+                              Headers to add the response prior to sending the response back to the client.
+                            item_type: !ruby/object:Api::Type::NestedObject
+                              properties:
+                                - !ruby/object:Api::Type::String
+                                  name: 'headerName'
+                                  description: |
+                                    The name of the header to add.
+                                - !ruby/object:Api::Type::String
+                                  name: 'headerValue'
+                                  description: |
+                                    The value of the header to add.
+                                - !ruby/object:Api::Type::Boolean
+                                  name: 'replace'
+                                  description: |
+                                    If false, headerValue is appended to any values that already exist for the header.
+                                    If true, headerValue is set for the header, discarding any values that were set for that header.
+                                  default_value: false
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'urlRewrite'
+                  description: |
+                    The spec to modify the URL of the request, prior to forwarding the request to the matched service.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'pathPrefixRewrite'
+                      description: |
+                        Prior to forwarding the request to the selected backend service, the matching portion of the
+                        request's path is replaced by pathPrefixRewrite.
+
+                        The value must be between 1 and 1024 characters.
+                    - !ruby/object:Api::Type::String
+                      name: 'hostRewrite'
+                      description: |
+                        Prior to forwarding the request to the selected service, the request's host header is replaced
+                        with contents of hostRewrite.
+
+                        The value must be between 1 and 255 characters.
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'timeout'
+                  description: |
+                    Specifies the timeout for the selected route. Timeout is computed from the time the request has been
+                    fully processed (i.e. end-of-stream) up until the response has been completely processed. Timeout includes all retries.
+
+                    If not specified, will use the largest timeout among all backend services associated with the route.
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'seconds'
```

Comment:
should this be an int like nanos?

Reply:
The api lists it as `string (int64 format)` and i saw another field similar use string for that.  I assumed `TypeInt` is just `int` and not `int64`, but I may have assumed wrong. And now I'm curious if I can use `TypeInt` for that?

Reply:
weird - well, if it works!